### ISSUE TITLE
Feature request: extra new installation screen that shows Computer ID

### ISSUE THREAD

**Issue by ianchanning** [hbons/SparkleShare#1880]
* Fedora 28
* SparkleShare v3.28 (flatpak)

See #1879 for current system, but I don't think this is related to a platform version, this is purely a UX feature request.

As a general note - I'm back using SparkleShare as a general solution instead of Dropbox because Dropbox has stopped support for some [full disk encrypted Linux file shares][1]:

> ecryptfs is not supported, but Dropbox will continue to sync with supported file systems that are encrypted via full disk encryption (e.g. LUKS)

### What happened:
1. First time install on a new system
2. Get initial screen asking for Name and Email
3. Get Sync Remote Project... screen
4. Sync fails because the current Computer ID doesn't exist on the server

### What I expected to happen:
In between asking for Name and Email add a screen that shows the current Computer ID in plain text and suggest adding it to the remote project server.

This issue is further compounded because on Wayland the Computer ID 'Copy to clipboard' currently doesn't work as per #1862.

### This happens when:
First time installation on a new computer

[1]: https://www.dropbox.com/help/desktop-web/system-requirements#linux

**Comment by qywx** [comment]
I met the same issue. How to resolve it manually?

**Comment by ianchanning** [comment]
@qywx You can get the Computer ID from the tray icon dropdown menu: 

![screenshot from 2018-11-11 16-10-17](https://user-images.githubusercontent.com/233318/48314639-7b542000-e5cc-11e8-90ed-234b11af2ac9.png)

**Comment by qywx** [comment]
@ianchanning  Thanks. I already found it.

**Comment by hbons** [comment]
You're totally right. I've got a fix for this but it still needs to be merged. See https://twitter.com/hbons/status/1033693191067852801
